src/main/java/org/folio/service/impl/EcsTlrServiceImpl.java

```diff
@@ -41,9 +43,16 @@ public EcsTlr create(EcsTlr ecsTlr) {
     log.debug("create:: parameters ecsTlr: {}", () -> ecsTlr);
     final String instanceId = ecsTlr.getInstanceId();
 
-    return tenantPickingStrategy.pickTenant(instanceId)
-      .map(tenantId -> createRequest(ecsTlr, tenantId))
-      .orElseThrow(() -> new TenantPickingException("Failed to pick tenant for instance " + instanceId));
+    List<String> tenantIds = tenantPickingStrategy.findTenants(instanceId);
+    for (String tenantId : tenantIds) {
+      try {
+        return createRequest(ecsTlr, tenantId);
+      } catch (TenantScopedExecutionException e) {
+        log.error("create:: cannot create a request for tenantId: {}", tenantId);
+      }
+    }
+    log.error("create:: failed to pick tenant for instance: {}", instanceId);
+    throw new TenantPickingException("Failed to pick tenant for instance " + instanceId);
```

Comment:
This does not make sense any more. We can reach this line even if we found tenants, but failed to create request in them